examples/custom_theme_example/widgetbook/main.widgetbook.dart

```diff
@@ -44,9 +44,7 @@ class HotReload extends StatelessWidget {
           devices: devices,
         ),
         CustomThemeAddon<AppThemeData>(
-          setting: CustomThemeSetting.firstAsSelected(
-            themes: [widgetbookTheme, widgetbookTheme2],
-          ),
+          themes: [widgetbookTheme, widgetbookTheme2],
```

Comment:
this code is not the same. In the old code, we have the widgetbookTheme as the default, here it's not clear (might be the same in the end). But do we have an option to specify what's selected as the default?

Reply:
For now, the first "element" in the list, is always the default selected one.
But we can add an optional parameter for the initial element as you suggested to add more flexibility.